htmldocs/.htaccess

```diff
@@ -18,11 +18,11 @@ RewriteRule ^service-config.html$ charms-config.html [R=301,L]
 RewriteRule ^charm-store.html$ authors-charm-store.html [R=301,L]
 RewriteRule ^charm-quality.html$ authors-charm-quality.html [R=301,L]
 RewriteRule ^charm-tests.html$ authors-testing.html [R=301,L]
-RewriteRule ^subordinate-services.html$ authors-subordinate-services.html [R=301,L]
+RewriteRule ^subordinate-services.html$ authors-subordinate-applications.html [R=301,L]
+RewriteRule ^authors-subordinate-services.html$ authors-subordinate-applications.html [R=301,L]
```

Comment:
I'm not sure, it came up in my find and replace. I assumed when we were talking about setting up redirects that this is the location they were setup.